<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1589795633806' stop='1589795712213'>
    <name>Connection</name>
    <title>Connection</title>
    <test-cases>
        <test-case start='1589795633806' status='failed' stop='1589795671540'>
            <name>In-place with Free plan</name>
            <title>In-place with Free plan</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1589795646421' status='undefined'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place with Free plan' source='8883963e-83a8-4906-9b74-d68d1e7c39c2-attachment.png' type='image/png' size='4214'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: Cannot read property 'currentStep' of undefined</message>
                <stack-trace>TypeError: Cannot read property 'currentStep' of undefined
                    at Allure.Object.&lt;anonymous>.Allure.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/allure-js-commons/index.js:86:45)
                    at Reporter.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/jest-allure/dist/Reporter.js:60:21)
                    at addAttachment (/home/travis/build/Automattic/jetpack/tests/e2e/lib/setup-env.js:26:12)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1589795671540' status='failed' stop='1589795705750'>
            <name>In-place upgrading a plan from personal to premium</name>
            <title>In-place upgrading a plan from personal to premium</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1589795680614' status='undefined'>
                    <name>Can set a sandbox cookie</name>
                    <title>Can set a sandbox cookie</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place upgrading a plan from personal to premium' source='cc883c59-5782-4975-8de7-64c4389bd851-attachment.png' type='image/png' size='4214'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: Cannot read property 'currentStep' of undefined</message>
                <stack-trace>TypeError: Cannot read property 'currentStep' of undefined
                    at Allure.Object.&lt;anonymous>.Allure.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/allure-js-commons/index.js:86:45)
                    at Reporter.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/jest-allure/dist/Reporter.js:60:21)
                    at addAttachment (/home/travis/build/Automattic/jetpack/tests/e2e/lib/setup-env.js:26:12)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>